docs/resources/lens/index.md

```diff
@@ -137,6 +137,12 @@ After creating a Lens data model, the next step is to explore it—this means in
   <!-- - [Supported Parameters for Tables and Views](/resources/lens/supported_parameters_for_tables_and_views/)
     Understand the role of logical tables and views in data models. -->
 
+## Supported data sources
+
+The Lens can be integrated with a variety of data sources. This section outlines the data sources supported by Lens and provides insights into connecting and managing data to ensure optimal performance and scalability.
+
+[Supported Data Sources](/resources/lens/data_sources/)
```

Comment:
You have to add the link of all the pages on the main page of Lens itself of all the supported sources, not create a sub page for it